### DISCUSSION TITLE
Form with labels left to each input

### DISCUSSION THREAD

**Discussion by andrisi** [quasarframework/quasar#14829] [General - Components / Directives / etc] [answered]
What's the best practice to create the "boring" sort of form, with input fields under each other, and labels to the left of each, not combined into the field? I see this: https://quasar.dev/vue-components/field#introduction but all of the examples are based on this fancy inline-label, that shrinks when the input is filled, and at most an iceon to the left. Great, but I'd want something simpler, just a label to the left...

![image](https://user-images.githubusercontent.com/1255809/200183722-84882de6-7be9-4da8-be56-90512f0bb285.png)

**Comment by suleimanas** [comment, marked as answer]
you can use slots - like **before**:
```
<q-input >
    <template #before>label</template>
</q-input>

```
